### ISSUE TITLE
Not entry in menu or shortcuts after installation in Windows 10 

### ISSUE THREAD

**Issue by LAfricain** [postiffm/bibledit-desktop#111]
All in the title, I created the shortcut navigating till the bibledit folder in Programms.

**Comment by postiffm** [comment]
We have to test this with the gtk3 branch, which has experienced important updates in this area.

**Comment by LAfricain** [comment]
Yes, Is it possible to merge already the new shortcut in master?

**Comment by rluzynski** [comment]
This should be verified first but indeed I think this can be (**partially!**) pushed to master because even although the changes are GTK3-specifis there is nothing that can't work in GTK2. Details below:

* ```New icons``` - because in the new *.desktop file I assume that the icons are located in the standard directories like ```/usr/share/icons/hicolor/96x96/apps```
* ```Do not use and do not distribute the old icon``` - optionally because maybe we don't need the old icon any more
* ```Port to GtkApplication framework``` - this must be split and only the part which updates *.desktop should be pushed to master. Note that GtkApplication was introduced only in GTK 3.4.

Note that even although it will fix the problem in Ubuntu it has nothing to do with Windows.

**Comment by postiffm** [comment]
Which commit from the gtk3 branch are you thinking?

**Comment by LAfricain** [comment]
The commit about the shortcut. It solved the problem on linux but I have no idea on Windows. If you merge this I will update my ubuntu ppa.

**Comment by rluzynski** [comment]
@postiffm do you need a pull request for this?

**Comment by postiffm** [comment]
I don't need one, but I need some time to figure out exactly which commit on the gtk3 branch to cherry-pick into the master. Once that is done, then I can remove that commit from gtk3 and we should not get any conflicts.

**Comment by rluzynski** [comment]
> I don't need one, but I need some time to figure out exactly which commit on the gtk3 branch to cherry-pick into the master.

It's explained in my comment above: https://github.com/postiffm/bibledit-desktop/issues/111#issuecomment-519517904. In case of the ```Port to GtkApplication...``` commit it's probably better to create a new commit using excerpts from the original one.

> Once that is done, then I can remove that commit from gtk3

They will be removed automatically when you rebase.

**Comment by LAfricain** [comment]
Solved with the last deb package for Ubuntu 18.04.
